lib/common.sh

```diff
@@ -253,11 +253,11 @@ export KIND_VERSION=${KIND_VERSION:-"v0.11.1"}
 if [ "${KUBERNETES_VERSION}" == "v1.21.2" ]; then
   export KIND_NODE_IMAGE_VERSION="v1.21.2"
   # Minikube version (if EPHEMERAL_CLUSTER=minikube)
-  export MINIKUBE_VERSION=${MINIKUBE_VERSION:-"v1.22.0"}
+  export MINIKUBE_VERSION=${MINIKUBE_VERSION:-"v1.25.1"}
 else
   export KIND_NODE_IMAGE_VERSION=${KIND_NODE_IMAGE_VERSION:-"v1.22.2"}
   # Minikube version (if EPHEMERAL_CLUSTER=minikube)
-  export MINIKUBE_VERSION=${MINIKUBE_VERSION:-"v1.23.2"}
+  export MINIKUBE_VERSION=${MINIKUBE_VERSION:-"v1.25.1"}
```

Comment:
`MINIKUBE_VERSION` updated by renovate is same in both `KUBERNETES_VERSION` s which is not correct, this has to be taken care of manually.